mimic_2_fate.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "70954479-ed2d-4bdb-90ca-5d607019e5b9",
```

Comment:
<p>Should be double checked if the original description is still up to date regarding cluster numbers?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
who should I ask here @Zethson ?

Reply:
Me? idk^^ the dataset is simple and we just look at it ourselves. Did the numbers look wrong, change or are fishy?

Reply:
they look fishy to me, and I wonder if the leiden clusters changed from when this text was written

Reply:
Totally possible. But unfortunately, it'd be you (us) that has to look at them. No expert involved.